sdk-core/src/main/java/care/data4life/sdk/RecordService.kt

```diff
@@ -1185,11 +1187,13 @@ class RecordService(
         const val FULL_ATTACHMENT_ID_POS = 1
         const val PREVIEW_ID_POS = 2
         const val THUMBNAIL_ID_POS = 3
+        // ToDo refactor
```

Comment:
if you use TODO the IDE is able to pick it up and present it in a project wide list. Just go `View` -> `Tool Windows` -> `TODO`

Reply:
You are totally right!